InputDataDirectory: '$(find vfh_recognition)/data'
InputDataExtension: '.pcd'
InputCloudTopic: '/camera/depth_registered/points'
Recognition:
  ServiceName: '/object_recognition'
  HistogramSize: 300
  SimilarityThreshold: 100.0
  NumberOfNeighbors: 1
  NormalEstimationRadius: 0.03
Segmentation:
  LeafSize:
    X: 0.01
    Y: 0.01
    Z: 0.01
  SpatialFilter:
    MinX: -0.7
    MaxX: 0.7
    MinY: 10.0
    MaxY: -10.0
    MinZ: 0.0
    MaxZ: 2.0
  ClusterConfiguration:
    Tolerance: 0.02
    MinSize: 100
    MaxSize: 25000
    
